### ISSUE TITLE
Support clojure 1.10.0

### ISSUE THREAD

**Issue by tony824** [clojurewerkz/elastisch#266]
in clojure 1.10.0, for `ns ` when `import`  , it is mandatory to specify at least one `class`
https://github.com/clojure/core.specs.alpha/blob/master/src/main/clojure/clojure/core/specs/alpha.clj#L140

we might have to change a couple of them 
 e.g.
https://github.com/clojurewerkz/elastisch/blob/master/src/clojurewerkz/elastisch/native/index.clj#L19

**Comment by Jarzka** [comment]
I noticed this problem too when I tried to upgrade to Clojure 1.10. I got the following error:

```
Syntax error macroexpanding clojure.core/ns at (index.clj:15:1).
() - failed: Insufficient input at: [:ns-clauses :import :classes :package-list :classes] spec: :clojure.core.specs.alpha/package-list
[org.elasticsearch.action.admin.indices.forcemerge.ForceMergeResponse] - failed: simple-symbol? at: [:ns-clauses :import :classes :class] spec: :clojure.core.specs.alpha/ns-import
:import - failed: #{:refer-clojure} at: [:ns-clauses :refer-clojure :clause] spec: :clojure.core.specs.alpha/ns-refer-clojure
:import - failed: #{:require} at: [:ns-clauses :require :clause] spec: :clojure.core.specs.alpha/ns-require
:import - failed: #{:use} at: [:ns-clauses :use :clause] spec: :clojure.core.specs.alpha/ns-use
:import - failed: #{:refer} at: [:ns-clauses :refer :clause] spec: :clojure.core.specs.alpha/ns-refer
:import - failed: #{:load} at: [:ns-clauses :load :clause] spec: :clojure.core.specs.alpha/ns-load
:import - failed: #{:gen-class} at: [:ns-clauses :gen-class :clause] spec: :clojure.core.specs.alpha/ns-gen-class
```

**Comment by Jarzka** [comment]
Any news on the update release?

**Comment by michaelklishin** [comment]
I have tagged `3.0.0` and released it but never pushed the tag or posted an update here. Sorry. `3.0.0` should be up on Clojars.

**Comment by rlovtangen** [comment]
> I have tagged `3.0.0` and released it but never pushed the tag or posted an update here. Sorry. `3.0.0` should be up on Clojars.

Wait, we have been using 3.0.0 for a year now. It was released on clojars.org on Dec 26, 2017. The new version with this fix should probably be 3.0.1 or something?

<img width="1127" alt="pasted_image_06_03_2019__11_04" src="https://user-images.githubusercontent.com/233839/53873288-531b8180-4000-11e9-854c-1b47679d887c.png">

**Comment by Jarzka** [comment]
Why is this issue closed since it has not been fixed?
